tests/integration/api_v3/http/posValidators.test.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+
+const { badRequestSchema } = require('../../../schemas/httpGenerics.schema');
+const { validatorsResponseSchema } = require('../../../schemas/api_v3/posValidators.schema');
+
+const endpoint = `${baseUrlV3}/pos/validators`;
+
+describe('pos/validators API', () => {
+	let numberActiveValidators;
+	let numberStandbyValidators;
+	let refGenerators;
+
+	beforeAll(async () => {
+		const response = await api.get(`${baseUrlV3}/pos/constants`);
+		numberActiveValidators = response.data.numberActiveValidators;
+		numberStandbyValidators = response.data.numberStandbyValidators;
+
+		const generatorsResponse = await api.get(`${baseUrlV3}/generators?limit=10`);
+		refGenerators = generatorsResponse.data;
+	});
+
+	describe(`GET ${endpoint}`, () => {
```

Comment:
Don't we already have test for validators?

Reply:
I could not find one. Hence added newly.